spec/support/ems_folder_helper.rb

```diff
@@ -0,0 +1,6 @@
+def vmware_nested_folders(ems)
```

Comment:
Please put this in a Module like `Spec::Support::ProviderStructures` rather than defining it on Object.  Inside your `describe` block you can `include Spec::Support::ProviderStructures`